packages/server/modules/workspaces/tests/integration/repositories.spec.ts

```diff
@@ -281,4 +283,61 @@ describe('Workspace repositories', () => {
       await expectToThrow(() => upsertWorkspaceRole(role))
     })
   })
+
+  describe.only('getWorkspaceRolesCountFactory creates a function, that', () => {
```

Comment:
describe.only

If you want to only run a specific block, use the IDE test explorer buttons or run the test w/ `yarn test --grep="filter"`